app/views/schools/register_mentor_wizard/check_answers.html.erb

```diff
@@ -52,7 +52,7 @@
     end
   end
 
-  if @mentor.provider_led_ect? && @mentor.mentoring_at_new_school_only?
+  if @mentor.provider_led_ect? && @mentor.mentoring_at_new_school_only? && @mentor.funding_available?
```

Comment:
It feels like there's quite a lot of business logic wrapped up in this template, perhaps once we're done with this journey we can look at reorganising `mentor.rb` a bit (perhaps splitting it up) so the logic is easy to get at from a testing POV?